code/game/objects/structures/disaster_counter.dm

```diff
@@ -30,16 +31,19 @@
 	SSpersistent_data.register(src)
 
 /obj/structure/disaster_counter/ex_act(severity)
-	if(severity < 2)
-		current_count = -1
-		persistent_save()
+	current_count = -1
+	persistent_save()
+	update_maptext()
 	. = ..()
 
 /obj/structure/disaster_counter/Destroy()
 	if(counter_id)
 		SSpersistent_data.registered_atoms -= src // Take us out the list
 	return ..()
 
+/obj/structure/disaster_counter/proc/update_maptext()
+	maptext = "<span class='maptext' style='text-align: center'>[max(current_count, 0)] / [record_count]</span>"
```

Comment:
```suggestion
	maptext = "<span class='maptext' style='text-align: center'>[max(current_count, 0)]/[record_count]</span>"
```
This spacing has an ingame effect. Its a string, not a mathematical operation.

Reply:
Yeah I realise, it just looked better to me with spacing between the numbers.
![NSS Cyberiad 2021-03-01 200435](https://user-images.githubusercontent.com/57483089/109552320-8565f980-7ac9-11eb-92ff-559c9d6830a7.png)
![NSS Cyberiad 2021-03-01 200423](https://user-images.githubusercontent.com/57483089/109552324-86972680-7ac9-11eb-8578-d254f78763ed.png)

Reply:
Try bumping the values to bigger numbers like 42 on each side and see how it looks

Reply:
![NSS Cyberiad 2021-03-01 203009](https://user-images.githubusercontent.com/57483089/109555245-3326d780-7acd-11eb-8e9e-025924aef965.png)

It only really starts clipping out of the screen when it goes above five digits, but fair point.